motoko/cert-var/src/cert_var_assets/src/index.js

```diff
@@ -0,0 +1,53 @@
+import { Actor, HttpAgent } from '@dfinity/agent';
+import { Certificate } from '@dfinity/agent';
+import { cert_var } from '../../declarations'
+
+document.getElementById("certifyBtn").addEventListener("click", async () => {
+  const newVal = BigInt(document.getElementById("newValue").value);
+  const _ = await cert_var.set(newVal);
+  const resp = await cert_var.get();
+
+  const agent = await Promise.resolve(new HttpAgent({ host: 'http://127.0.0.1:' + port, identity })).then(
```

Comment:
@chenyan-dfinity How do I determine `port` here?  Should I hardcode a port to go with the hardcoded URL, like 8080?

Reply:
Oh, but I also need to know `identity` as well.